src/shared/ui/modal/modal.module.scss

```diff
@@ -0,0 +1,76 @@
+.modal {
+  max-width: 889px;
+  padding: 0 32px 32px 32px;
+
+  line-height: 24px;
+
+  border: none;
+  border-radius: 12px;
+
+  &::backdrop {
+    background-color: hsl(0deg 0% 0% / 50%);
+  }
+
+  @include media-tablet {
+    padding: 0 16px 16px 16px;
+  }
+
+  @include media-tablet-large {
+    max-width: revert;
+  }
+}
+
+.modal-content {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+.modal-header {
+  position: sticky;
+  top: 0;
+
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  padding: 26px 0 24px 0;
```

Comment:
I suggest to remove padding-bottom here - it seams to me it will be looks better

Reply:
In that case, some other modal could look like this:
![image](https://github.com/user-attachments/assets/0f920817-933e-441c-84ef-a67ed35f6360)
So I'd stick with the current design for now

Reply:
Sorry, I keep forgetting that we have the mockup now ))))